### DISCUSSION TITLE
[Done] InvenTree Community Survey 0.6.0 Release / Jan 2022

### DISCUSSION THREAD

**Discussion by matmair** [inventree/InvenTree#2561] [User survey] [answered]
Help us understand your deployments and needs.
The survey is anonymous and takes roughly 3-5 minutes.
https://s.surveyplanet.com/y0dw92gg

**Comment by huseyinkogo** [comment]
done :)

**Comment by matmair** [comment]
So the release did take a little bit longer than Jan 2022 but do not worry: The results will be released in 2 weeks so ppl. Have time to fill it out once they update.

**Comment by matmair** [comment, marked as answer]
The survey has concluded, the results will be shared shortly in a separate repo:
https://github.com/inventree/inventree_survey/tree/main/0_Jan2022

There will be infographics together with the most of the raw data.
